andela/ah-frontend-thanos
# #161348757 Users should be able to view and edit their  pro…

#### What does this PR do?
Users can edit and view their profiles

#### Description of Task to be completed?
- View a user's Bio, image( if any)
- Users can edit only their profiles that is first name, last name, bio, image

#### How should this be manually tested?
- Go to the article's URL: `/profiles/:username` e.g. `/profiles/rachael`
- You should be able to see your profile
- You should be able to edit your profile

#### Any background context you want to provide?
You do need to be logged in to view your profile
#### What are the relevant pivotal tracker stories?
[#161348757](https://www.pivotaltracker.com/story/show/161348757)

#### Relevant Screenshots
<img width="1415" alt="screen shot 2018-12-12 at 10 40 08 pm" src="https://user-images.githubusercontent.com/22786444/49894482-1572e680-fe5f-11e8-99d9-9e434870f913.png">
<img width="1407" alt="screen shot 2018-12-12 at 10 40 43 pm" src="https://user-images.githubusercontent.com/22786444/49894493-1ad03100-fe5f-11e8-9b1b-f5388e3fdb87.png">
